Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM geospaas:latest
+LABEL purpose="testing ´api´s for Django-Geo-SpaaS"
+ENV PYTHONUNBUFFERED=1
+
+# Install Django
+RUN pip install \
+    djangorestframework \
+    markdown \
+    django-filter 
+
```

Comment:
Don't add so many empty lines without a need.

Reply:
It is ok.